qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -0,0 +1,171 @@
+import logging
+from StringIO import StringIO
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+
+class XFSTestsDev(CephFSTestCase):
+    def setUp(self):
+        CephFSTestCase.setUp(self)
+        self.prepare_xfstests_dev()
+
+    def prepare_xfstests_dev(self):
+        self.get_repo()
+        self.get_test_and_scratch_dirs_ready()
+        self.install_deps()
+
+        # NOTE: On teuthology machines it's necessary to run "make" as
+        # superuser since the repo is cloned somewhere in /tmp.
+        self.mount_a.client_remote.run(args=['sudo', 'make'],
+                                       cwd=self.repo_path, stdout=StringIO(),
+                                       stderr=StringIO())
```

Comment:
On teuthology machines it requires superuser access to operate on directories in /tmp.

Reply:
> make install almost certainly does though. That said, I generally never call make install with xfstests and just run it out of the build tree, so YMMV.

The make install part isn't clear to me. Can you please elaborate?

Reply:
You're doing a "make" and then a "make install". The "make" typically doesn't require any special privileges since you're just operating in the build dir, but the "make install" can drop files all over the filesystem, so you often need root privileges for that.

You can also run xfstests directly out of the tree without running "make install", but now that I look, I don't see directions for that in the README.

Reply:
> You can also run xfstests directly out of the tree without running "make install", but now that I look, I don't see directions for that in the README.

Would be nice we could run without doing "make install".